ui/src/dashboard/menus/MenuCategoryForm.tsx

```diff
@@ -3,25 +3,39 @@ import { FieldArray } from 'formik'
 import { Field, Button } from 'components'
 import { MenuItemForm } from './MenuItemForm'
 import { MoneyField } from 'components/forms/MoneyField'
+import { Dropzone } from 'components/Dropzone'
 
-export function MenuCategoryForm({ values, initialCategory, initialItem }) {
+export function MenuCategoryForm({ values, initialCategory, initialItem, setFieldValue }) {
   return (
     <FieldArray name="categories">
       {({ insert, remove, push }) => (
         <div className="card card--full-width card--lightestGray">
           {values.categories.length > 0 &&
             values.categories.map((category, index) => (
               <div key={index} className="card card--full-width card--line margins">
-                <Field
-                  isFormik
-                  name={`categories.${index}.name`}
-                  placeholder="Appetizers"
-                  type="text"
-                  label="Category Name"
+                <div className="catergory-container grid">
+                  <div className="left">
+                    <Field
+                      isFormik
+                      name={`categories.${index}.name`}
+                      placeholder="Appetizers"
+                      type="text"
+                      label="Category Name"
+                      // TODO Update Formik on change
```

Comment:
I think this comment needs moved down as it is for the MoneyField, not for this appetizer Field